package.json

```diff
@@ -15,19 +15,21 @@
   },
   "scripts": {
     "clean": "rm -rf dist/ && rm -rf coverage/",
-    "build": "tsc  --p  tsconfig.build.json\n",
     "prebuild": "npm run clean",
+    "build": "tsc --p tsconfig.build.json",
+    "build:watch": "tsc -w --p tsconfig.build.json",
     "test": "cd ./test && npm run test",
     "start": "ts-node ./src/server/tests/app.ts",
     "pretest": "npm run lint",
     "install-test": "cd ./test && npm i",
     "lint": "eslint 'src/**/*.ts' --fix",
     "copy-files": "cp package.json ./dist/package.json & cp README.md ./dist/README.md",
-    "publishh": "npm run build && npm version patch --force && npm run copy-files && cd dist && npm publish && cd ../ && npm run clean"
+    "publish": "npm run build && npm version patch --force && npm run copy-files && cd dist && npm publish && cd ../ && npm run clean"
```

Comment:
The command "publish" is already npm command, so we must rename it.

Reply:
Ah you're right. I just no-brained to "typo" and must have "corrected it" by mistake.